src/Campaigns/Actions/EnqueueCampaignPageEditorAssets.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+use Give\Campaigns\Models\CampaignPage;
+use Give\Framework\Support\Facades\Scripts\ScriptAsset;
+
+/**
+ * @unreleased
+ */
+class EnqueueCampaignPageEditorAssets
+{
+    /**
+     * @unreleased
+     */
+    public function __invoke()
+    {
+        $currentPost = get_post();
+
+        if($currentPost->post_type !== 'give_campaign_page') {
+            return;
+        }
+
+        $campaignPage = CampaignPage::find($currentPost->ID);
```

Comment:
Models are nullable so we should return early if not found to avoid fatal errors trying to access property on null.

Reply:
In this situation, the `CampaignPage` model represents the post being edited, so it would have to exist.

Probably still worth handling here, so an early return is probably sufficient.